DESCRIPTION

```diff
@@ -10,14 +10,18 @@ Authors@R: c(person("Bioconductor Package Maintainer", role = c("aut", "cre"),
 	      person("Valerie", "Obenchain", role = "aut"),
 	      person("Michael", "Love", role = "aut"),
 	      person("Lori", "Shepherd", role = "aut"),
-	      person("Martin", "Morgan", role = "aut"))
+	      person("Martin", "Morgan", role = "aut"),
+              person("Sonali", "Kumari", role = "ctb",
+              comment = "Converted GinomicFles vignette from GenomicFiles.Rnw to RMarkdown / HTML."),
```

Comment:
Indent line 15. It should be indented like line 9.

Correct spelling of `GenomicFiles`. We typically use the following for the comment: `Converted 'GenomicFiles' vignettes from Sweave to RMarkdown / HTML.`

Reply:
ohk